npm/vue/package.json

```diff
@@ -8,24 +8,24 @@
     "cy:open": "node ../../scripts/cypress.js open --component --project ${PWD}",
     "cy:run": "node ../../scripts/cypress.js run --component --project ${PWD}",
     "build": "rimraf dist && rollup -c rollup.config.js",
-    "postbuild": "node ../../scripts/sync-exported-npm-with-cli.js",
-    "typecheck": "vue-tsc --noEmit",
+    "postbuild": "node --require @packages/ts/register ./inline-types.ts && node ../../scripts/sync-exported-npm-with-cli.js",
+    "typecheck": "yarn tsd && vue-tsc --noEmit",
     "test": "yarn cy:run",
+    "tsd": "yarn build && yarn tsc -p test-dts/tsconfig.tsd.json",
```

Comment:
I assume `tsd` stands for "typescript definitions", what is `dts` in the directory name meant to represent?

Reply:
Good question, I actually have no idea, I copied it from this code base I work on a lot, where someone else set it up: https://github.com/vuejs/test-utils/tree/main/test-dts

I think `dts` is `.d.ts` as in a `foo.d.ts`, the definitions for `foo`.

I will change `test-dts` to `test-tsd` to be consistent.

Reply:
Done